dds/DCPS/NetworkConfigModifier.h

```diff
@@ -10,8 +10,6 @@
 
 #include "ace/config.h"
 
-#if defined(ACE_HAS_GETIFADDRS) && !defined(OPENDDS_SAFETY_PROFILE) && !defined (ACE_LINUX)
-
```

Comment:
This should be restored probably. Safety profile build is failing. Also ACE treats Android a separate platform as far as `ACE_LINUX` goes if that's why you removed it.

Reply:
So [`ACE_LACKS_GETIFADDRS` is being defined for APIs < 24](https://github.com/DOCGroup/ACE_TAO/blob/78006062ece6be12eba3e81471961c74743ba071/ACE/ace/config-android.h#L74-L76).

Reply:
Had to do a workaround for this as ACE is not properly setting ACE_HAS_GETIFADDERS for android.